omnigibson/robots/manipulation_robot.py

```diff
@@ -279,6 +279,7 @@ def _find_gripper_contacts(self, arm="default", return_contact_positions=False):
 
         # Get robot links
         link_paths = set(self.link_prim_paths)
+        finger_paths = set([link.prim_path for link in self.finger_links[arm]])
 
         if not return_contact_positions:
```

Comment:
Error we should fix, showing up on tests below:

(We should try to find a way to NOT raise an error, live with it somehow) 

```
2024-09-27T03:21:20.1766869Z Traceback (most recent call last):
2024-09-27T03:21:20.1768882Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/simulator.py", line 324, in <lambda>
2024-09-27T03:21:20.1770478Z     lambda _: self._on_physics_step()
2024-09-27T03:21:20.1771825Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/simulator.py", line 1114, in _on_physics_step
2024-09-27T03:21:20.1772885Z     obj.step()
2024-09-27T03:21:20.1773974Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/objects/controllable_object.py", line 446, in step
2024-09-27T03:21:20.1775205Z     self.deploy_control(control=u_vec, control_type=u_type_vec)
2024-09-27T03:21:20.1776623Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/robots/manipulation_robot.py", line 364, in deploy_control
2024-09-27T03:21:20.1777765Z     self._handle_assisted_grasping()
2024-09-27T03:21:20.1779128Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/robots/manipulation_robot.py", line 1344, in _handle_assisted_grasping
2024-09-27T03:21:20.1780584Z     self._establish_grasp(arm=arm, ag_data=self._calculate_in_hand_object(arm=arm))
2024-09-27T03:21:20.1782117Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/robots/manipulation_robot.py", line 1375, in _establish_grasp
2024-09-27T03:21:20.1783534Z     return self._establish_grasp_rigid(arm, ag_data, contact_pos)
2024-09-27T03:21:20.1785046Z   File "/_work/default_slurm_1/OmniGibson/OmniGibson/omnigibson-src/omnigibson/robots/manipulation_robot.py", line 1253, in _establish_grasp_rigid
2024-09-27T03:21:20.1786265Z     assert contact_pos is not None, (
2024-09-27T03:21:20.1788215Z AssertionError: contact_pos in self._find_gripper_contacts(return_contact_positions=True) is not found in self._find_gripper_contacts(return_contact_positions=False). This is likely because GripperRigidContactAPI.get_contact_pairs and get_contact_data return inconsistent results.
```